frontend/src/renderer/components/Process/FileRoles.svelte

```diff
@@ -0,0 +1,157 @@
+<script lang="ts">
+    import { processState } from "@util/stores";
+    import Dropbox from "@components/Process/Dropbox.svelte";
+    import {get, isEmpty, each, includes} from "lodash";
+    import { autoSortImages } from "@util/autoSortStandards.svelte";
+
+    let imageStack = get($processState, 'artStacks[0].fields');
+    let rerenderToggle = false;
+
+    // this helps force a rerender once the imageStack has been reset
+    $:if ($processState.currentTab === 0) {
+        imageStack = get($processState, 'artStacks[0].fields');
+    }
+
+    const getAllImages = function () {
+        let allImages = [];
+        each(imageStack, function (field) {
+            if(!isEmpty(field)) {
+                allImages.push(field[0]);
+            }
+        });
+        each($processState.imageFilePaths, function (image) {
+            if(!includes(allImages, image)) {
+                allImages.push(image);
+            }
+        });
+        return allImages;
+    };
+
+    const showTargetDropZones = function () {
+        return getAllImages().length > 6;
+    };
+
+    const autoSort = function () {
+        $processState.imageFilePaths = autoSortImages(getAllImages(), imageStack);
+        rerenderToggle = !rerenderToggle;
+    };
+
+    const resetImages = function () {
```

Comment:
How are we handling this when batch processing comes in? This should be non hard coded if possible

Reply:
huh, this seems more complex than it should be

Reply:
I'm not even getting the 'reset images' button:
- [ ] fix reset images button

<img width="451" alt="Screen Shot 2023-02-14 at 3 33 01 PM" src="https://user-images.githubusercontent.com/43387387/218855624-ec9782a2-55f7-4379-b259-8984af8811d7.png">

Reply:
> How are we handling this when batch processing comes in? This should be non hard coded if possible

This is now fixed

Reply:
Are you sure it is fixed? It looks like the old version is just the current version, but unwrapped. What happens in the case of extra images?

Reply:
![image](https://user-images.githubusercontent.com/15817820/219095281-5228bbf0-6414-4079-9321-d0265ce2984b.png)
This is what I see when I run locally and all changes are pushed.  This is in a smaller window.